client.Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION} AS builder
+ARG PKGVER
+RUN pip wheel --wheel-dir=/wheels --no-cache-dir backend.ai-client==${PKGVER}
+
+FROM python:${PYTHON_VERSION}
+COPY --from=builder /wheels /wheels
+RUN mkdir -p /root/.ssh
+RUN apt-get update && apt-get install -y vim
```

Comment:
It is recommended to clear apt caches after installing apt packages. Ref: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run